addons/event_sale/models/event.py

```diff
@@ -41,6 +43,17 @@ class Event(models.Model):
         'event.event.ticket', 'event_id', string='Event Ticket',
         copy=True)
 
+    sale_order_lines_ids = fields.One2many(
+        'sale.order.line', 'event_id',
+        string='All sale order lines pointing to this event')
+
+    sales_total_price = fields.Monetary(compute='_compute_sales_total_price')
```

Comment:
Just for coherency, I think most sale related fields are named sale_something, not sales (like sale_avg_price in product for example; it matches sale_order or sale_order_line)